INSTALL_MACOS.md

```diff
@@ -35,48 +34,60 @@ Homebrew is easier to set up than MacPorts, but cannot build universal binaries.
 
 You need an up to date MacPorts version. Just download it and install it from <http://www.macports.org>, and execute the following commands in a terminal:
 
-    sudo port selfupdate
-    sudo port upgrade outdated
+```shell
+sudo port selfupdate
+sudo port upgrade outdated
+```
 
 Then, you should add the "ports" provided by Natron. Edit as root the file `/opt/local/etc/macports/sources.conf` (as in `sudo nano /opt/local/etc/macports/sources.conf`) and add the following line at the beginning, with the path to the Natron sources (yes, there are three slashes after `file:`):
 
-    file:///Users/your_username/path_to_sources/Natron/tools/MacPorts
+```
+file:///Users/your_username/path_to_sources/Natron/tools/MacPorts
+```
 
 Then, create the index file:
 
-    (cd /Users/your_username/path_to_sources/Natron/tools/MacPorts; portindex)
+```
+(cd /Users/your_username/path_to_sources/Natron/tools/MacPorts; portindex)
+```
 
 It is also recommended to add the  following line to `/opt/local/etc/macports/variants.conf`:
 
-    -x11 +no_x11 +bash_completion +no_gnome +quartz +natron
+```
+-x11 +no_x11 +bash_completion +no_gnome +quartz +natron
+```
 
-#### Notes for OS X 10.6 Snow Leopard
```

Comment:
This is actually Mac OS X Snow Leopard https://en.wikipedia.org/wiki/Mac_OS_X_Snow_Leopard 
Don't blame me, blame Apple for the name changes!

Reply:
Oh... I do ;P